add README mentioning that this package is obsolete now

dirk-thomas opened the pull request:
Connect to ament/ament_python#3.

mikaelarguedas commented:
The title doesn't match the code change, maybe some uncommited changes?

If the goal of this PR is what is mentionned in the title, I'd suggest emptying this repo leaving only the README with the obsolete notice (like we've done for other now-obsolete repos recently (gmock_vendor, gtest_vendor, ceres-solver))

dirk-thomas commented:
> The title doesn't match the code change, maybe some uncommited changes?

Yeah, thanks for checking. I certainly forgot to add the new README. No idea how I got to these weird changes in the `setup.py` file though :worried:

I fixed the commit and added a second one to empty the repo (except the README).
